.github/workflows/graphene_tests.yml

```diff
@@ -25,22 +25,22 @@ jobs:
       GRAPHENE_OPT_CMD: opt-${{ matrix.llvm-version }}
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-      with:
-        show-progress: false
-    - uses: actions/setup-python@v5
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install hatch
-      run: pip install hatch==1.9.4
-    - name: Install clang and lli
-      run: |
-        wget https://apt.llvm.org/llvm.sh
-        chmod +x llvm.sh
-        sudo ./llvm.sh ${{ matrix.llvm-version }}
-        sudo apt-get install -y llvm-${{ matrix.llvm-version }}-runtime
-    - name: Fetch tags
-      run: git fetch --tags --force origin
-    - name: Run tests
-      run: hatch run dev:test
+      - name: Checkout
```

Comment:
Any reason for the indentation changes? I'm not really sure whats considered good yml tbh

Reply:
Ugh no, just vscode decided that this looks better now. Does yours prefer the old indentation?